src/mod_generics_cache.py

```diff
@@ -1,14 +1,53 @@
 """Module for testing the behavior of generics across different modules."""
```

Comment:
I only added a bunch of variable annotations here but since this wasn't compatible with < 3.5, I had to uglify this module. Now it's actually easier to see what's going on just by looking at the entire file with the "View" button.